k3s/scorpio/scorpio.yaml

```diff
@@ -0,0 +1,120 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgis
+  labels:
+    app.kubernetes.io/name: postgis
+    app.kubernetes.io/instance: postgis
+spec:
+  type: ClusterIP
+  ports:
+    - port: 5432
+      targetPort: 5432
+      protocol: TCP
+      name: postgis
+  selector:
+    app.kubernetes.io/name: postgis
+    app.kubernetes.io/instance: postgis
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: postgis
+  labels:
+    app.kubernetes.io/name: postgis
+    app.kubernetes.io/instance: postgis
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: postgis
+      app.kubernetes.io/instance: postgis
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: postgis
+        app.kubernetes.io/instance: postgis
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: postgis
+          imagePullPolicy: IfNotPresent
+          # as by scropio doc, no version provided
+          image: "postgis/postgis"
+          ports:
+            - name: http
+              containerPort: 5432
+              protocol: TCP
+          env:
+            - name: POSTGRES_USER
+              value: "ngb"
+            - name: POSTGRES_PASSWORD
+              value: "ngb"
+            - name: POSTGRES_DB
+              value: "ngb"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: scorpio
+  labels:
+    app.kubernetes.io/name: scorpio
+    app.kubernetes.io/instance: scorpio
+spec:
+  type: LoadBalancer
+  ports:
+    # same as orion, makes other config easier
+    - port: 1026
+      targetPort: 9090
+      protocol: TCP
+      name: scorpio
+  selector:
+    app.kubernetes.io/name: scorpio
+    app.kubernetes.io/instance: scorpio
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: scorpio
+  labels:
+    app.kubernetes.io/name: scorpio
+    app.kubernetes.io/instance: scorpio
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: scorpio
+      app.kubernetes.io/instance: scorpio
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: scorpio
+        app.kubernetes.io/instance: scorpio
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: scorpio
+          imagePullPolicy: IfNotPresent
+          image: "scorpiobroker/all-in-one-runner:java-kafka-4.1.4"
+          env:
+            - name: DBHOST
+              value: postgis
+            - name: BUSHOST
```

Comment:
Is that all thats necessary to integrate Kafka too? Nice :D

Reply:
In scorpio yes, but the Kafka deployment itself is in the other yaml file;)

Reply:
Yea I saw that, lovely slim and easy deployment :sweat_smile:

Reply:
Will scorpio read from Kafka too?

Reply:
Yes. Scorpio (in contrast to Orion-LD) is built in a microservice architecture and uses kafka as the bus(similar to the kiwi-bus;)) between the individual services. Its normally not only one container, but they also publish the "aio" container, that holds all of the services in a single container. I used this one, to keep the setup more readable.

Reply:
kiwi-bus :face_exhaling: Not making this better :laughing: 
How are Kafka and the Blockchain going to work together? "uncritical" entities on kafka and orders&co on blockchain?

Reply:
There is no blockchain here. The kafka is just for inter-service communication inside scorpio. The blockchain in dome is completely seperated from that.